CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Possible log types:
 - [changed] Use [serde flatten](https://serde.rs/attr-flatten.html) to
   implement extensions. This changes the in memory representation of the
   extensions to include the `ext_` prefix. (#79)
+- [added] Add missing stream field. (#80)
```

Comment:
Nitpicking: I'd remove the period unless the changelog entry consists of multiple full sentences (like the one above).

Reply:
Fixed.